src/bench/mempool_stress.cpp

```diff
@@ -66,15 +66,17 @@ static void ComplexMemPool(benchmark::Bench& bench)
                 tx.vin.back().scriptSig = CScript() << coin.tx_count;
```

Comment:
on line 59 above, should Available also be made a reference? Otherwise the modifications to the coin would not be recorded since it's a copy?

Reply:
Oops My bad  , yes Available needs to be a reference , fixing it changes the benchmark results substantially . 

|               ns/op |                op/s |    err% |     total | benchmark
|--------------------:|--------------------:|--------:|----------:|:----------
|    1,103,652,315.00 |                0.91 |    2.6% |     13.34 | `ComplexMemPool`